CMakeLists.txt

```diff
@@ -187,8 +211,8 @@ if(DPKG_PROG)
   set(CPACK_SYSTEM_NAME ${CPACK_DEBIAN_PACKAGE_ARCHITECTURE})
 endif()
 # Debian versions require a dash
```

Comment:
Can you remove this line?

Reply:
done: https://github.com/frankaemika/libfranka/pull/86/commits/54c777abca34eb829a6d2a3e7446aff6581ea68a
Though now that I think about it, according to the [cmake docs](https://cmake.org/cmake/help/v3.6/module/CPackDeb.html) `CPACK_DEBIAN_PACKAGE_ARCHITECTURE` is populated by the output of `dpkg --print-architecture` *by default*, so all that if block above seems redundant.
If you agree, I can refactor that too though I cannot test it in a debian-based distro for the moment.

Reply:
Good point.. I tested it and it seems that this is indeed not required, so the block can be removed.

Reply:
I removed them here: https://github.com/frankaemika/libfranka/pull/86/commits/fede7423c88f4f54f208dcdf40429a788c71aa52